### DISCUSSION TITLE
AppKit Support

### DISCUSSION THREAD

**Discussion by jeromelaban** [unoplatform/uno#13811] [Q&A]
_From @gameDNAstudio: in https://github.com/unoplatform/uno/pull/11116#issuecomment-1734474175:_

Any update on this ?

**Comment by jeromelaban** [comment]
@gameDNAstudio not at this time, we're still building for AppKit, but the support from the .NET Runtime is not complete enough to be viable at this time. Is catalyst a potential target for you?

**Reply by gameDNA-Helix** [reply]
As a temporary workaround yes but I prefer AppKit.

**Reply by jeromelaban** [reply]
Got it, thanks. What's in AppKit that's of interest, out of curiosity?
